crates/agglayer/src/main.rs

```diff
@@ -40,13 +40,13 @@ fn main() -> anyhow::Result<()> {
         }
         cli::Commands::Vkey => {
             let vkey = agglayer_prover::compute_program_vkey(ELF);
-            println!("{}", vkey);
+            println!("0x{}", vkey.trim_start_matches("0x"));
         }
 
         cli::Commands::VkeySelector => {
-            let vkey_selector =
+            let vkey_selector_hex =
                 hex::encode(pessimistic_proof::core::PESSIMISTIC_PROOF_PROGRAM_SELECTOR);
-            println!("{vkey_selector}");
+            println!("0x{}", vkey_selector_hex.trim_start_matches("0x"));
```

Comment:
Even more consistency could be achieved by changing the type of `pessimistic_proof::core::PESSIMISTIC_PROOF_PROGRAM_SELECTOR` to `alloy_primitives::B32`. That way, the `0x` prefix will be added anywhere it's printed out of the box.

Even better, we'd have a dedicated type for vkey selectors instead of plain bytes.

Reply:
up to you! it was just a suggestion :)